man/corsix-th.6.md

```diff
@@ -0,0 +1,56 @@
+% corsix-th(6) | Games Manual
+%
+% 2018-08-13
+
+# NAME
+
+corsix-th - A Theme Hospital engine reimplementation.
+
+# SYNOPSIS
+
+**corsix-th** [--interpreter=*luafile*] [--config-file *configfile*] [--load= *savefile*] [--dump=strings]
+
+# DESCRIPTION
+
+Theme Hospital was a simulation computer game developed by Bullfrog
+Productions and published by Electronic Arts in 1997, in which the player
+designs and operates a hospital.
+
+This project aims to reimplement the game engine
+of Theme Hospital, and be able to load the original
+game data files. This means that you will need a
+purchased copy of Theme Hospital, or a copy of the
+demo, in order to use CorsixTH.
+
+On Debian systems these data files can be
```

Comment:
Consult the readme file for details how to make the game data of Theme Hospital available for the program.